src/utils/hibp.js

```diff
@@ -146,8 +146,13 @@ async function loadBreachesIntoApp (app) {
       // sync the "breaches" table with the latest from HIBP
       await upsertBreaches(breaches)
     }
-    const breachLogoMap = await downloadBreachIcons(breaches)
-    app.locals.breachLogoMap = breachLogoMap
+    downloadBreachIcons(breaches).then(breachLogoMap => {
+      app.locals.breachLogoMap = breachLogoMap
+    })
+    // This will be replaced by a map with the breach logos when
+    // `downloadBreachIcons` resolves, but by setting it to an empty Map first,
+    // we don't delay the server start - we just won't have breach logos yet.
+    app.locals.breachLogoMap = new Map()
```

Comment:
Would it be better to put this line above 149, to avoid the potential of race condition 149 already executed (chance is low yes) and `breachLogoMap` is already set and now it's reset